conf/ocsci/openshift-console/openshift_console_no_failfast.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# To avoid skipping remaining portion of tests upon encountering the first failure
+RUN:
+  openshift_console_no_failfast: true
```

Comment:
s/openshift_console_no_failfast/openshift_console_failfast/  we dont have to negate a boolean variable.

Reply:
Variable used in openshift console has exact this name: NO_FAILFAST so I am keeping their convention.

I think that openshift console test execution fails after first test failed. Wit this env variable set, the openshift console will continue with other tests. @ebondare asked me to allow this option and I've created this configuration file for it.

@ebondare am I right?

Reply:
@petr-balogh yes, that's true, thank you